full-moon/src/ast/mod.rs

```diff
@@ -260,6 +261,31 @@ impl Default for TableConstructor<'_> {
     }
 }
 
+impl Sealed for TableConstructor<'_> {}
+impl<'a> NodeTrait<'a> for TableConstructor<'a> {
```

Comment:
We have a separate file for these implementations instead of mod.rs, which is already pretty beefy.